docker-contents/bootstrap.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Does this script still work with current directory structure?

Reply:
Haha, I was not even aware of those scripts as I was too concentracted on the autobuild on Docker Hub. They have now been updated to work after the merge. The docker-contents/README.rst had been updated as well.